Library/Homebrew/cmd/certs.rb

```diff
@@ -0,0 +1,50 @@
+#:  * `certs` [`--force-curl`]:
+#:    You should really write more documentation up here Dom.
+
+module Homebrew
+  BREWED_CURL = Pathname.new(HOMEBREW_PREFIX/"opt/curl")
+  BREWED_OPENSSL = Pathname.new(HOMEBREW_PREFIX/"opt/openssl")
+  BREWED_LIBRESSL = Pathname.new(HOMEBREW_PREFIX/"opt/libressl")
+  BREWED_GNUTLS = Pathname.new(HOMEBREW_PREFIX/"opt/gnutls")
+
```

Comment:
Yeah, I don't have much of a desire to create a command that updates every cert, only really the OpenSSL/LibreSSL/GnuTLS ones because we create them.

I think it's reasonable to presume that if we create them we'll provide a easy method of keeping them refreshed, but beyond that we don't police packages.